### ISSUE TITLE
User Preferences - Exit Option

### ISSUE THREAD

**Issue by b-k-white** [tsgrp/HPI#1164]
Currently, when you enter the user preferences module, it is impossible to exit without refreshing your page. The user preferences should have an "Exit" button, or allow the button to toggle preferences instead of just open them.

**Comment by gsteimer** [comment]
Wouldn't a browser refresh just reload the user preferences page again?  Not sure what you mean by a browser refresh getting the user back to the page they were on...

**Comment by b-k-white** [comment]
I guess back works - I wasn't aware that user preferences had its own URL. I wasn't expecting that I was losing my current context when clicking on user preferences, but given that knowledge, making the user preferences a toggle doesn't seem necessary. We could probably close this issue if you agree @gsteimer.

**Comment by gsteimer** [comment]
Yeah, I agree - the Back button would probably be your best bet with the current code.  I'll mark this UCR in case a client really wants us to refactor this at some point.  Seems unlikely though.
